sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java

```diff
@@ -866,7 +866,18 @@ private void completeAllSinks() {
         if (completeEmitResult == Sinks.EmitResult.OK) {
             logger.debug("Main sink completed, Context: {}", this.operationContextText);
         } else {
-            logger.info("Main sink completion failed. EmitResult: {}, Context: {}", completeEmitResult, this.operationContextText);
+            if (completeEmitResult == Sinks.EmitResult.FAIL_CANCELLED ||
+                completeEmitResult == Sinks.EmitResult.FAIL_TERMINATED) {
+
+                logger.debug("Main sink already completed, EmitResult: {}, Context: {}",
+                    completeEmitResult,
+                    this.operationContextText);
+            } else {
+                logger.info(
```

Comment:
Should this be info? I assume this is not a success state, so why not `warn` or `error` (if it is actionable) ?

Reply:
I think with moving the high volumn logs to debug we can make this warning